packages/fxa-settings/pm2.config.js

```diff
@@ -63,5 +63,11 @@ module.exports = {
       min_uptime: '2m',
       time: true,
     },
+    {
+      name: 'gql-whitelist',
+      autorestart: false,
+      script: 'yarn gql:whitelist',
+      watch: ['src/**/*.ts'],
```

Comment:
Does `watch` work when `autorestart` is false?  And does the server need to be restarted to load the updated whitelist?

Reply:
Ah good call. I think this slipped through the cracks when I finalizing the changes. Let me test this again. Perhaps it's half baked.

Reply:
I just got done validating this. Turns out that autorestart false does work best with the watch. There's other examples of this already in place, which this was initially copied from. It turns out that without it the watch process can continually restarted leading to an odd behavior.